ibis/pandas/tests/test_operations.py

```diff
@@ -904,3 +904,20 @@ def test_cast_to_decimal(t, df, type):
         1 <= len(element.as_tuple().digits) <= type.precision
         for element in result.values
     )
+
+
+@pytest.mark.parametrize(
+    ('ibis_func', 'pandas_func'),
+    [
+        (lambda x: x.clip(lower=0), lambda x: x.clip(lower=0)),
+        (lambda x: x.clip(lower=0.0), lambda x: x.clip(lower=0.0)),
+        (lambda x: x.clip(upper=0), lambda x: x.clip(upper=0)),
+        (lambda x: x.clip(lower=0, upper=1), lambda x: x.clip(lower=0, upper=1)),
+        (lambda x: x.clip(lower=0, upper=1.0), lambda x: x.clip(lower=0, upper=1.0)),
+        (methodcaller('quantile', 0.5), methodcaller('quantile', 0.5)),
+    ]
+)
+def test_arrayfunctions(t, df, ibis_func, pandas_func):
```

Comment:
I would just have a separate test that only contains failures (using `with pytest.raises`). Easier to debug that way.